spec/filters/kubernetes_metadata_spec.rb

```diff
@@ -0,0 +1,363 @@
+# base don https://github.com/logstash-plugins/logstash-output-http/blob/master/spec/outputs/http_spec.rb
+
+require "spec_helper"
+require "logstash/filters/kubernetes_metadata"
+require "logstash/json"
+require "sinatra"
+require "thread"
+
+PORT = rand(65535-1024) + 1025
+
+
+class TestApp < Sinatra::Base
```

Comment:
I do too! I would definitely like to add more test coverage in the future... Thanks. :)